src/app/utils/utils.ts

```diff
@@ -47,17 +47,17 @@ const DAY_MILLIS = 24 * HOUR_MILLIS;
 // [     0,    1,    2,    3     ] array
 //       0     1     2     3       indexes
 // {  0  |  1  |  2  |  3  |  4  } gap indices (drop zones)
-export function moveDashboardCard(arr: CardConfig[], from: number, gapIndex: number) {
+export function moveDashboardCard(cards: CardConfig[], from: number, gapIndex: number) {
```

Comment:
This function and below seems to work for any array. We can generalize it by removing type assertions here so we don't have to import redux slices.